documentation/modules/auxiliary/scanner/mssql/mssql_login.md

```diff
@@ -0,0 +1,43 @@
+## Vulnerable Application
+
+This module simply queries the MSSQL instance for a specific user/pass (default is sa with blank). 
+
+## Verification Steps
+Example steps in this format (is also in the PR):
+
+1. Start msfconsole
+2. Do: ```use scanner/mssql/mssql_login```
+3. Do: ```set RHOSTS [IP]```
+4. Do: ```run```
```

Comment:
Should use single backquotes (`) for single line literals instead of triple backquotes (```)
```suggestion
2. Do: `use scanner/mssql/mssql_login`
3. Do: `set RHOSTS [IP]`
4. Do: `run`
```